mne_bids/read.py

```diff
@@ -331,18 +331,21 @@ def read_raw_bids(bids_basename, bids_root, modality=None, extra_params=None,
     sub = bids_basename.subject
     ses = bids_basename.session
 
+    # set root, infer the modality and
+    # then set it to the modality and suffix of the BIDSPath
+    bids_basename.update(root=bids_root)
```

Comment:
Should we change the signature of `read_raw_bids` to use `root` instead of `bids_root` too in this PR? Or is this reserved for a followup? Currently this seems a little inconsistent.

Reply:
No actually, we could / should drop `bids_root` here entirely in favor of `BIDSPath.root`